PyP100/PyP100.py

```diff
@@ -0,0 +1,354 @@
+import requests
+from requests import Session
+
+from base64 import b64encode, b64decode
+import hashlib
+from Crypto.PublicKey import RSA
+import time
+import json
+from Crypto.Cipher import AES, PKCS1_OAEP, PKCS1_v1_5
+from . import tp_link_cipher
+import ast
+import pkgutil
+import uuid
+import json
+
+
+#Old Functions to get device list from tplinkcloud
+def getToken(email, password):
+	URL = "https://eu-wap.tplinkcloud.com"
+	Payload = {
+		"method": "login",
+		"params": {
+			"appType": "Tapo_Ios",
+			"cloudUserName": email,
+			"cloudPassword": password,
+			"terminalUUID": "0A950402-7224-46EB-A450-7362CDB902A2"
+		}
+	}
+
+	return requests.post(URL, json=Payload).json()['result']['token']
+
+def getDeviceList(email, password):
```

Comment:
Great job on naming! The names are very clear and not too long. However, we are following pep8 conventions, so I think we should use snakecase in function names instead of camelcase.